include/class.forms.php

```diff
@@ -2732,10 +2730,8 @@ function hasIdValue() {
 
     function getChoices($verbose=false) {
         if (!isset($this->_choices)) {
-            $choices = array('' => '— '.__('Default').' —');
-            foreach ($this->getTopics() as $t)
-                $choices[$t->getId()] = $t->getName();
-            $this->_choices = $choices;
+            $this->_choices = array('' => '— '.__('Default').' —') +
```

Comment:
Just want to note that this default value doesn't actually choose the default topic, but that's part of what I'm working on in another PR. Other than that, looks good.

Reply:
Dully noted, thanks